draft-ietf-dnsop-7706bis.xml

```diff
@@ -93,7 +93,9 @@
       resolver for good queries on TLDs, because the TTL for most TLDs is
       usually long-lived (on the order of a day or two) and is thus usually
       already in the cache of the recursive resolver; the same is true for the
-      TTL for negative answers from the root servers.</t>
+      TTL for negative answers from the root servers. (Although the primary
+      goal of the design is for serving the root zone, the method can be
+      used for any zone.)</t>
```

Comment:
Non-blocking nit - do whatever you want:
Personally I would have worded this as:
"... the root servers (although the..." or just made the "(Although the..." bit be not parenthetical.

Note that this is just an opinion, I'll merge the change, but my OCD needed me to comment...